dali/operators/numba_function/numba_func.cu

```diff
@@ -112,9 +112,41 @@ bool NumbaFuncImpl<GPUBackend>::SetupImpl(std::vector<OutputDesc> &output_desc,
   return true;
 }
 
+
+std::vector<void*> prepare_args(const dali::TensorList<dali::GPUBackend> &in, uint64_t *ptr, uint64_t shape_ptr) {
+  void** meminfo = new void*;
+  void** parent = new void*;
+  *meminfo = NULL;
+  *parent = NULL;
+  ssize_t item_size = dali::TypeTable::GetTypeInfo(in.type()).size();
+
+  vector<void*> args;
```

Comment:
I would add some comment with the reference why you insert each data in this particular order.

Reply:
Done